src/MainState/Behavior.cpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * @file Behavior.cpp
+ * @brief behavior class correlating names, ids, priorities
+ *
+ * @author Connor Novak
+ * @email connor@students.olin.edu
+ * @date 2018-11-21
+ * @version 1.0.0
+ */
+
+#include "Behavior.h"
+
+/*
+ * @brief Constructor TODO(connor@students) Turn into struct
+ */
+Behavior::Behavior(const char* name, const int label)
+ : name(name), id(label) { }
+
+int Behavior::getId() {
+  return id;
+}
```

Comment:
In the interest of time, you do not need to add this. However, in the future, use a static variable which holds the already taken ids and throw an exception when the specified id is taken.

Reply:
#91